lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -2172,7 +2221,8 @@ export class UserAgentApplication {
         }
         // if set then validate
         const { applicationName, applicationVersion, telemetryEmitter } = config;
-        if (!applicationName || !applicationVersion || ! telemetryEmitter) {
+        if (!applicationName || !applicationVersion || !telemetryEmitter) {
+            console.log("Am I here");
```

Comment:
Guessing this can be removed?

Reply:
Done. Missed checking in removing this, added for tests.